doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -640,6 +640,7 @@ Modem libraries
 
 * :ref:`pdn_readme` library:
 
+  * Added ``PDN_EVENT_APN_RATE_CONTROL_ON`` and ``PDN_EVENT_APN_RATE_CONTROL_OFF`` events to report on the status of APN rate control.
```

Comment:
I believe we can link to these new events using :c:enumerator:`EVENT_NAME`.

Reply:
Thanks for pointing this Pekka. I tried, :c:enumerator:`PDN_EVENT_APN_RATE_CONTROL_ON` and :c:enumerator:`PDN_EVENT_APN_RATE_CONTROL_OFF`, and it correctly links to the corresponding API docs.

Reply:
nice, thanks